components.html

```diff
@@ -0,0 +1,403 @@
+                        </nav>
+                    </div>
+                    <div class="component-nav2">
+                        <nav>
+                            <img src="/images/dribbble-4.svg" alt="">
+                            <div class="nav2-list">
+                                <ol>
+                                    <li>Inspiration</li>
+                                    <li>Find Work</li>
+                                    <li>Learn Design</li>
+                                    <li>Go Pro</li>
+                                    <li>Hire Designers</li>
+                                </ol>
+                            </div>
+                            <div class="nav1-user">
+                                <button class="nav1-user-account upload-btn">
+                                    Upload
+                                </button>
+                            </div>
+                        </nav>
+                    </div>
+                </div>
+            </div>
+            <div class="component-card">
+                <h2 class="component-sample-heading" id="heading-alert">ALERTS</h2>
+                <div class="components-grid2">
+                   <div class="component-alert1">
+                       <div class="alert-success">
+                           <p>Hey! Your Tansaction has been Successfully Made!</p>
+                       </div>
+                   </div>
+                   <div class="component-alert2">
+                       <div class="alert-failure">
+                           <p>Oops! an error encountered. Please try after some time!</p>
+                       </div>
+                   </div>
+                    <div class="component-alert3">
+                        <div class="alert-success-animate">
+                            <img src="https://www.freeiconspng.com/thumbs/success-icon/success-icon-10.png" alt="">
+                            <div class="alert-success-animate-text">
+                                <h3>Success!</h3>
+                                <p>The process was successfull</p>
+                                <button>close</button>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="component-alert4">
+                        <div class="alert-failure-animate">
+                            <img src="/images/kisspng-exclamation-mark-clip-art-vector-graphics-computer-5ced299834b624.4021236515590465522159-removebg-preview.png" alt="">
+                            <div class="alert-failure-animate-text">
+                                <h3>Oops!</h3>
+                                <p>Error occured</p>
+                                <button>close</button>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="component-alert5">
+                        <div class="toast-success">
```

Comment:
Why toast component under alert component? I didn't understand the reason behind this maybe look into this.